src/components/ResizeHandler/ResizeHandler.js

```diff
@@ -99,26 +99,40 @@ class ResizeHandler extends PureComponent {
       document.documentElement.style.setProperty(stylePropHeight, `${vh}px`);
     }
 
-    if (onResize) {
-      onResize(entries, maxWidthBrkpts, maxHeightBrkpts);
-    }
-
     if (!maxWidthBrkpts && !maxHeightBrkpts) {
+      onResize(entries);
       return;
     }
 
+    let newScreenWidth;
+    let newScreenHeight;
+
     for (let i = 0; i < entries.length; i += 1) {
       const entry = entries[i];
       const rect = entry.contentRect;
       const { height, width } = rect;
 
       if (maxWidthBrkpts) {
-        ResizeHandler.applyBreakpoints(entry, maxWidthBrkpts, width, 'w');
+        newScreenWidth = ResizeHandler.applyBreakpoints(
```

Comment:
applyBreakpoints returns no value

Reply:
Indeed, I updated with the latest version from geops-react-ui